pkg/manifests/bundleloader.go

```diff
@@ -49,33 +49,58 @@ func (b *bundleLoader) LoadBundle() error {
 
 // Compress the bundle to check its size
 func (b *bundleLoader) calculateCompressedBundleSize() error {
+	if b.bundle == nil {
+		return nil
+	}
 	err := filepath.Walk(b.dir,
 		func(path string, info os.FileInfo, err error) error {
 			if err != nil {
 				return err
 			}
+			if info.IsDir() {
+				return nil
+			}
+			data, err := os.ReadFile(path)
+			if err == nil {
+				fileSize, err := fileSize(path)
+				if err != nil {
+					return err
+				}
+				contentGzip, err := encoding.GzipBase64Encode(data)
+				if err != nil {
+					return err
+				}
+				// Sum the compressed amount
+				totalCompressed := int64(len(contentGzip))
+				if b.bundle.CompressedSize != nil {
+					totalCompressed += *b.bundle.CompressedSize
+				}
+				b.bundle.CompressedSize = &totalCompressed
 
-			if !info.IsDir() {
-				if data, err := os.ReadFile(path); err == nil {
-					content, err := encoding.GzipBase64Encode(data)
-					if err != nil {
-						return err
-					}
-					total := int64(len(content))
-					if b.bundle.CompressedSize != nil {
-						total += *b.bundle.CompressedSize
-					}
-					b.bundle.CompressedSize = &total
+				// Sum the bundle amount
+				totalSize := fileSize
+				if b.bundle.Size != nil {
```

Comment:
Move this above the compressed size calculation so its closer to where we read the uncompressed file size?

Reply:
done